data/noiseprofiles.json

```diff
@@ -3043,7 +3043,22 @@
             {"name": "E-M10MarkII iso 12800", "iso": 12800, "a": [0.000481645929928644, 0.00019163744252251, 0.000362326729576843], "b": [2.38746195987496e-06, 2.2508533772738e-06, 2.68981097675674e-06]},
             {"name": "E-M10MarkII iso 25600", "iso": 25600, "a": [0.000993794684356688, 0.000404854394329772, 0.000759301032073675], "b": [5.40625299417711e-06, 5.72061095738384e-06, 6.53536538330311e-06]}
           ]
+        },
+        {
+          "comment": "Olympus OM-D E-M1 Mk.II profiled by Dawid Loubser (dawid.loubser@ibi.co.za)",
+          "model": "E-M1MarkII",
+          "profiles": [
+            {"name": "E-M1MarkII iso 64", "iso": 64, "a": [6.03448971452101e-06, 3.34702089057382e-06, 7.96405397670638e-06], "b": [5.64107208449212e-09, -2.5739502568846e-09, 3.54038442599048e-09]},
+            {"name": "E-M1MarkII iso 200", "iso": 200, "a": [6.13709922433744e-06, 3.54055600967237e-06, 8.11208952906179e-06], "b": [8.96247561553474e-09, 5.87516750118224e-09, 7.9760196090394e-09]},
+            {"name": "E-M1MarkII iso 400", "iso": 400, "a": [1.1277313652014e-05, 6.30341903686042e-06, 1.41886067124167e-05], "b": [1.54555312163237e-08, 7.06721019161393e-09, 1.48892314882207e-08]},
+            {"name": "E-M1MarkII iso 800", "iso": 800, "a": [2.11892459998804e-05, 1.12541173528155e-05, 2.56595516307882e-05], "b": [4.04565499925071e-08, 2.69246669230953e-08, 4.18664327519833e-08]},
+            {"name": "E-M1MarkII iso 1600", "iso": 1600, "a": [4.22953887838241e-05, 2.14948866100995e-05, 4.90974948586661e-05], "b": [8.22142492924679e-08, 6.15912238619032e-08, 8.30371607130872e-08]},
+            {"name": "E-M1MarkII iso 3200", "iso": 3200, "a": [8.09094889300661e-05, 4.03657735201826e-05, 9.11000593033239e-05], "b": [1.54850201636701e-07, 1.06491270486505e-07, 1.4547431044408e-07]},
+            {"name": "E-M1MarkII iso 6400", "iso": 6400, "a": [0.000162818243811292, 7.66465569572085e-05, 0.000189807939134868], "b": [3.4511459701267e-07, 2.72253170538817e-07, 3.08685445026257e-07]},
+            {"name": "E-M1MarkII iso 12800", "iso": 12800, "a": [0.000341007611560922, 0.000170115910815632, 0.000400188562023328], "b": [7.56753327460077e-07, 7.05251299439448e-07, 6.39453192395373e-07]}
+          ]
         }
+
```

Comment:
please remove extra newline. Afterward PR can be merges, noise profile looks ok.